packages/compass/src/app/components/update-toasts.tsx

```diff
@@ -61,37 +77,66 @@ const RestartCompassToastContent = ({
   );
 };
 
-export function onAutoupdateStarted() {
-  openToast('update-download', {
+const UpdateInstalledToastContent = ({
+  newVersion,
+}: {
+  newVersion: string;
+}) => {
+  const darkmode = useDarkMode();
+  return (
+    <div className={containerStyles}>
+      <Body className={textStyles}>
+        Compass {newVersion} is installed successfully
+      </Body>
+      <Link
+        as="a"
+        className={cx(linkStyles, darkmode && linkDarkStyles)}
+        href={`https://github.com/mongodb-js/compass/releases/tag/v${newVersion}`}
+      >
+        Release notes
+      </Link>
+    </div>
+  );
+};
+
+export function onAutoupdateStarted({ newVersion }: { newVersion: string }) {
+  openToast('compass-update-started', {
     variant: 'progress',
-    title: 'Compass update is in progress',
+    title: `Compass ${newVersion} is downloading`,
   });
 }
 export function onAutoupdateFailed() {
-  openToast('update-download', {
+  openToast('compass-update-failed', {
     variant: 'warning',
     title: 'Failed to download Compass update',
     description: 'Downloading a newer Compass version failed',
   });
 }
 export function onAutoupdateSuccess({
-  updatedVersion,
+  newVersion,
   onUpdate,
   onDismiss,
 }: {
-  updatedVersion: string;
+  newVersion: string;
   onUpdate: () => void;
   onDismiss: () => void;
 }) {
-  openToast('update-download', {
-    variant: 'note',
+  openToast('compass-update-succeeded', {
+    variant: 'success',
     title: '',
     description: (
       <RestartCompassToastContent
-        updatedVersion={updatedVersion}
+        newVersion={newVersion}
         onUpdateClicked={onUpdate}
       />
     ),
     onClose: onDismiss,
   });
 }
+export function onAutoupdateInstalled({ newVersion }: { newVersion: string }) {
+  openToast('compass-update-restarted', {
+    variant: 'note',
```

Comment:
This seems to be success in the designs, not note

Reply:
updated in [a19f91d](https://github.com/mongodb-js/compass/pull/5718/commits/a19f91d28aec1aa5d06f6f394ffeb955f6410b73)